DO NOT MERGE - Just for comparison of latest updates

Robbie1977 opened the pull request.

matentzn commented:
All our changes were merged into the official OLS release. I created another PR to see what will happen if we just use that one..

henrietteharmse committed:
Uncommented @Ignore in ApiDocumentation.java

simonjupp committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS into 3.x-dev

simonjupp committed:
prefer english labels as primary

henrietteharmse committed:
This commit includes small changes that aims to address the following:
(1) Fixes for the null-pointer exception and deadlock that slipped in as
part of the implementation of ticket #241.
(2) To pinpoint the errors encountered in (1), logging and exception
handling has been improve slightly.
(3) Ticket #236 has been address by not allowing any exception error
messages to propagate to the front-end.
(4) I introduced a typo in the API docs that I fixed.
(5) Ticket #241 still has not been fixed.

henrietteharmse committed:
Fix for ticket #241 is now working.

henrietteharmse committed:
Reverted MongoDB fix for now as it is too extreme. I.e., we still want
for 404 errors to display sensible error message.

henrietteharmse committed:
A less extreme fix for ticket #236 that will give a `A server error
occurred` only for HTTP 500 status errors.

simonjupp committed:
class renderer will handle when no language tag is present fixes #261

henrietteharmse committed:
Merge pull request #262 from EBISPOT/261

class renderer will handle when no language tag is present fixes #261

henrietteharmse committed:
Upgraded to JUnit 5 to allow for using parameterised tests.

henrietteharmse committed:
This commit includes 2 main changes:
(1) Replaced log4j.xml with logback-test.xml since the code base is in
actual fact using logback rather than log4j.
(2) Loading of preferredRootTerms from an ontology has been added.

simonjupp committed:
added docker file for Solr and docker-compose for dev envrionment

henrietteharmse committed:
Initial steps towards being able to unit test Neo4J indexing.

henrietteharmse committed:
Initial testing and refactor of Neo4J.

henrietteharmse committed:
Neo4J indexes updated for preferred root terms. Includes:

(1) Implementation
(2) Simple integration test

henrietteharmse committed:
Removed preferred roots that was accidentally added Solr TermDocument.

henrietteharmse committed:
Potential fix for Mondo issue

henrietteharmse committed:
Merge branch '3.x-dev'

simonjupp committed:
Merge branch '3.x-dev' into dockercompose

warrenread committed:
Resurrect Thymeleaf integration for new branding; dockerise solr & mongo

Hard-incorporate Tommy's existing modified templates (from branch
new-ui) over old-style html (with equivalent functionality), also adding
new banner image. Fix a problem with Thymeleaf and self-closing tags.

Implement dockerised solr and mongo instances: add appropriate
dockerfile for OLS, plus bash scripts resprectively to create, and to
run, the necessary docker image.

Note that this is mostly untested: not only does the UI still require
work, but functionality might also be broken.

henrietteharmse committed:
Fixed issue with cycles in properties not being detected.

henrietteharmse committed:
Fix for detecting cycles in properties.
Merge branch '3.x-dev'

henrietteharmse committed:
Added Spring health check

henrietteharmse committed:
Merge branch '3.x-dev' with Spring health check enabled.

henrietteharmse committed:
Partial preferred roots implementation with new ols tree components. Displaying roots is working. What is not working consistently is displaying a specific term.

simonjupp committed:
Merge branch 'dockercompose' into 3.x-dev

simonjupp committed:
added an API endpoint at api/ols-config that returns config in yaml. You can also filter on ontology ids using the ids parameter fixes #275

simonjupp committed:
added an API endpoint at api/ols-config that returns config in yaml. You can also filter on ontology ids using the ids parameter fixes #275

simonjupp committed:
fixed formatting error so config is under a ontologies property in the yaml

simonjupp committed:
Merge pull request #276 from EBISPOT/config_export

config export endpoint fixes #275

warrenread committed:
Conditional styling in header; update CSS; generalise OLS docker image

Enable conditional (parameterisable) placement of typeahead CSS into
Thymeleaf inclusion, and pass in parameter from several templates.
Include and update OLS-specific CSS. Include jsTree CSS. Rename
duplicate element id in Thymeleaf inclusion, to prevent whitespace
appearing around banner.

Add new dockerfile for OLS, parameterisable and runnable by branch:
different branches can now be run in parallel on the same host. Add
matching run and build scripts.

Update readme.

warrenread committed:
Merge branch '3.x-dev' into ebi-rebranded

henrietteharmse committed:
Neo4j queries not fully implemented yet, nor working properly. Commit to allow merge of new branding.

henrietteharmse committed:
Added architecture diagram.

henrietteharmse committed:
Added architecture diagram.

warrenread committed:
Correct final search page record count (fix #279); fix 'Prev' navigation

warrenread committed:
Remove commented code line, for brevity ease of diffing

warrenread committed:
Fix search and index page layout with CSS and template modifications

Recolour buttons and filter box. Give buttons round corners. Left-align
search page elements. Enable fieldset to collapse inside containing div.
Reduce vertical whitespace between search results. Fix top checkbox
spacing. Fix facet text so it wraps, and facet box so it expands to
accommodate facets. Lose hard breaks and non-breaking spaces: spacing
issues are much better handled in CSS. Add extra classes where
necessary.

Fix indentation, spacing and other formatting issues in JavaScript code.

Remove redundant, commented code form search page template.

warrenread committed:
Merge branch '3.x-dev' into ebi-rebranded

warrenread committed:
Update readme to correct typos and trademarks

warrenread committed:
Incorporate typo and trademark corrections from ebi-rebranded branch

warrenread committed:
Retrofit (to modified HTML) of commit 964b6c by henrietteharmse

Original text: "A less extreme fix for ticket #236 that will give a 'A
server error occurred' only for HTTP 500 status errors".

warrenread committed:
Resolve conflicts

simonjupp committed:
Merge pull request #283 from EBISPOT/3.x-dev

3.x dev

warrenread committed:
Merge branch '3.x-dev' into ebi-rebranded (for diagram in readme)

warrenread committed:
Whiten Elixir footer background; modify cursor and colour on facet hover

Elixir background made white (modified from dark grey) by passing
data-color attribute from div to JavaScript function.

EBI's global Foundation-based CSS uses automatic cursor styling, which
has to be overridden for HTML buttons, which conventiontally retain the
default arrow cursor; this is because normally buttons have strong
styling cues of their own (contrastingly to in-text hyperlinks), but
that is not the case here. Implement background colour change
separately, through :hover and :focus pseudo-classes.

warrenread committed:
Update Cypher DSL repository URL in POM

henrietteharmse committed:
A working implementation of preferred roots

henrietteharmse committed:
Deleted integration test depending on Neo4j

henrietteharmse committed:
Refactor of JS components to use element names rather than ids

simonjupp committed:
Create .gitlab-ci.yml

henrietteharmse committed:
Merge 'preferred-roots' into 3.x-dev

henrietteharmse committed:
Fixed sibling label status issue.

henrietteharmse committed:
Fixed preferred roots issue causing null pointer exception on some reasoners

henrietteharmse committed:
Improved styling of preferred root radio buttons

henrietteharmse committed:
Preferred root terms can now be specified in YAML config.

henrietteharmse committed:
Show preferred root option only for ontologies with preferred roots configured

henrietteharmse committed:
Preferred roots displayed when found in Neo4j or Mongo

henrietteharmse committed:
Fixed preferred root state issue

johannes-darms committed:
Update YamlConfigParser.java

prevent NPE if no ontologies are defined.

warrenread committed:
Improve and ease runnability of OLS branches within Docker

Make it possible to extract, build and run dockerised OLS not only by
branch, but by individual Git commit. Add entrypoint script to run
services (specifically MongoDB and Solr, but optionally incuding Tomcat,
and by extension OLS itself) automatically. All services currently run
within a single container, without orchestration. Multiple versions of
OLS can nonetheless be run simulataneously, within different containers,
using different port mappings to expose service endpoints: to effect
this, an integer increment to the standard port numbers can simply be
passed to the run script as its final parameter; use different increment
values if calling the script more than once.

Add entrypoint script, volume delete script, supervisor config file and
optional addendum thereto (for Tomcat).

warrenread committed:
Fix id of main content area to correct behaviour and stop console errors

warrenread committed:
Fix accidental delegation, which was obfuscating facets' event handling

warrenread committed:
Remove box shadow around search strings; use GNU grep within container

warrenread committed:
Fix right-hand panel heading background colour, and depth (roughly)

warrenread committed:
Use appropriate icon for 'Term mappings'

warrenread committed:
Update readme for docker commit-specific local deployments

henrietteharmse committed:
Fix for Neo4j NullPointerException. Changed dubug logging to log stacktrace.

henrietteharmse committed:
Fixed accidental rename of obo-config.yaml

henrietteharmse committed:
Merge branch '3.x-dev' which implements preferred roots.

warrenread committed:
Implement Sass/SCSS pre-compilation; adjust alignment and font sizing

SCSS syntax stylesheets now reside in ols-web/src/main/resources/scss,
from which they pre-compile during the build phase and will overwrite
any equivalently-named css files in
ols-web/src/main/resources/static/css. Because SCSS is a superset of
CSS, ideally no individual file should be placed in the latter
directory, since any syntactically-valid CSS file can simply be renamed
with the extension "scss", and then placed in the former, with the
option of adding SCSS sugar later.

Align ontology links using some absolute-positioning trickery, which at
least ensures text runs along a shared baseline.

Expand breadcrumb font size (alignment is still an issue here).

warrenread committed:
Update Tomcat minor version for Docker

warrenread committed:
Fix breadcrumb vertical alignment and horizontal spacing

henrietteharmse committed:
Fixes for tickets #297 and #294

henrietteharmse committed:
Merged with 3.x-dev

# Conflicts:
#	README.md
#	ols-web/src/main/resources/static/documents/api.html
#	ols-web/src/main/resources/templates/individual.html
#	ols-web/src/main/resources/templates/ontology.html
#	ols-web/src/main/resources/templates/property.html
#	ols-web/src/main/resources/templates/term.html

henrietteharmse committed:
Added new_ols.css

henrietteharmse committed:
Fix for tree component issues for properties and individuals.

henrietteharmse committed:
Merge 3.x-dev into master

warrenread committed:
Enable all ontologies to be viewed on ontologies browse page

warrenread committed:
Restore short name coloured backround on ontologies list page

warrenread committed:
Adjust left margins to fix horizontal spacing; add SassDoc for mixin

warrenread committed:
Improve URI display in search results page

warrenread committed:
Cut template whitespace; tidy html; add spacer CSS; serve source map

Break term template lines so that they are a little more readable.
Effect elimination of whitespace by means of newline-spanning HTML
comments; this has the countervailing consequence of reducing
readabilty, but a modern IDE should deal with this fairly well. (There
is currently no recognised best practice way of eliminating whitespace
text nodes without comprising source readability slightly.)

To improve spacing in the term page, add new spacer class (elements
assigned to which should be left empty). Cut several redundant lines
from source SCSS.

Ensure generated source map is pushed to the same directory as output
CSS, ensuring SCSS source code can be accessed via browsers' development
tools.

warrenread committed:
Populate individuals page; adjust whitespace below header & above footer

warrenread committed:
Undo EBI styling's breadcrumb capitalisation; add icon-text interspace

warrenread committed:
Remove exclamation mark from search button

warrenread committed:
Restore tree icons denoting partonomic relations

warrenread committed:
Restyle and tooltip term URIs; add thinspace class; fix SCSS warnings

henrietteharmse committed:
Fixes the following small issues:
(1) Log file for config-importer changed to logback-spring.xml to adhere to Spring best practice.
(2) Fixed issue where when indexing failed for an ontology without fail status being written to mongo, the indexer did not re-index the ontology on subsequent runs.
(3) Fixed spelling error on loading report.
(4) Fixed issue where purl is not specified when overriding config will result NullPointerException.

henrietteharmse committed:
Change from logback.xml to logback-spring.xml according to Spring recommendation

henrietteharmse committed:
Merged 3.x-dev

henrietteharmse committed:
VersionIri's should rather not be displayed than displayed with empty string.

henrietteharmse committed:
Merge 3.x-dev

henrietteharmse committed:
Commented out gitlab config issue for now

warrenread committed:
Add protocol for using containerised Solr and MongoDB instances

warrenread committed:
Merge branch 'ebi-rebranded' into ebi-rebranded-merge2

warrenread committed:
Remove post-precompilation CSS; alter active search box highlight colour

warrenread committed:
Merge branch 'ebi-rebranded-merge' into ebi-rebranded-merge2

warrenread committed:
Enable execute permission on script

warrenread committed:
Add focus pseudo-class, and intensify emphasis around search box

warrenread committed:
Sort additional ontologies list ('Also in: ...') by prefix

warrenread committed:
Merge branch '3.x-dev' into ebi-rebranded-merge

warrenread committed:
Merge branch 'ebi-rebranded-merge' into ebi-rebranded-merge2

warrenread committed:
Fix whitespace and issue with broken (HTMLencoded) HTML comment syntax

henrietteharmse committed:
#306 Change preferred roots to use http://purl.obolibrary.org/obo/IAO_0000700.
#325 Property hierarchies across imports did not work.

henrietteharmse committed:
Deleted catalog-v001.xml file

henrietteharmse committed:
Merge 3.x-dev

warrenread committed:
Enable Foundation; Thymeleafise CSS links; tidy code; volumise Solr data

Update legacy Bootstrap HTML attributes to those required by Foundation
layout engine. Extend inclusion of CSS in Thymeleaf head fragment, and
remove hanging link tags between head and body: fixes rendering of
search dropdowns.

Reformat HTML to break multiple attributes across multiple lines, for
basic readability. Balance opening and closing tags to restore correct
formatting on some pages. Enforce better indentation hierarchy for HTML
tags, for code intelligibility. Compress commented-out code. Tidy up
actual text comments a bit.

Add Solr data volume to dockerised services startup script.

Add Javax Annotation API to POM.

warrenread committed:
Fix Foundation classes; fix icon; balance & align tags; improve styling

Assign "breadcrumbs" class to appropriate element. Place navigable
elements in defined list. Employ correct "copy" icon. Remove superflous
styling (which made no difference). Balance and align tags, for both
human and machine readability. Split closing inline tags across lines,
to avoid both spurious spacing and use of dummy HTML comments. Lose
superfluous inner HTML section. Minor styling change to text colour.
Tidy JavaScript and enforce use of semi-colon command endings.

Generally, make HTML consistent across property and term page; this
could still benefit from modularisation with Thymeleaf, or alternative
programmatic enforcement.

jamesamcl committed:
Fix font in tree to be consistent with the rest of the page

jamesamcl committed:
Fix icon spacing

jamesamcl committed:
Fix home button

jamesamcl committed:
Fix the term tree on the ontology page

jamesamcl committed:
Fix some colours, styling, and text (info -> information, id -> ID)

jamesamcl committed:
More style fixes and add ols.debrand option (#332)

henrietteharmse committed:
Added documentation about how to fix OLS build errors

henrietteharmse committed:
Merge branch '3.x-dev'

jamesamcl committed:
Logo and title customisation options (#332)

jamesamcl committed:
Hide docs and about for ols.debrand (#332)

jamesamcl committed:
Add customisation section to README

jamesamcl committed:
shortName -> shortTitle and fix "null"s in term page

jamesamcl committed:
Fix Term relations header styling

jamesamcl committed:
Top align text in ontology table

jamesamcl committed:
Update ebi:owner

jamesamcl committed:
Merge remote-tracking branch 'origin/3.x-dev' into ebi-rebranded-merge2

jamesamcl committed:
Make blue things green

jamesamcl committed:
Clickable links in annotations

jamesamcl committed:
Index data properties in solr

matentzn committed:
Create Makefile to run docker build commands

henrietteharmse committed:
.war Manifest fix

henrietteharmse committed:
.pom cleanup

jamesamcl committed:
Reinstate changes to file hashing after verified working on prod

jamesamcl committed:
Reinstate changes to file hashing after verified working on prod

jamesamcl committed:
Hopefully fix the issue of ontologies not updating

The hash of the ontology is now updated in the database, only AFTER the
ontology has been loaded successfully (unlike the download hash file
which is updated regardless).  If this hash does not match a downloaded
ontology, the ontology is reloaded.

jamesamcl committed:
Merge branch 'patch-1' of github.com:johannes-darms/OLS into 3.x-dev

jamesamcl committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS into ebi-rebranded-merge2

jamesamcl committed:
Merge branch '3.x-dev' into make-docker

jamesamcl committed:
Merge pull request #335 from EBISPOT/make-docker

Create Makefile to run docker build commands

matentzn committed:
Update Makefile

matentzn committed:
Merge branch 'make-docker' of https://github.com/EBISPOT/OLS into make-docker

jamesamcl committed:
enable static web content to be overridden from ols home

henrietteharmse committed:
Merge pull request #341 from EBISPOT/make-docker

Make docker merged into 3.x-dev

jamesamcl committed:
Implement allow_download ontology option

jamesamcl committed:
Implement allow_download ontology option

henrietteharmse committed:
Merge pull request #340 from EBISPOT/fix-updating-ontologies

Hopefully fix the issue of ontologies not updating

henrietteharmse committed:
Merge pull request #345 from EBISPOT/allow-download-old-ui

Implement allow_download ontology option

jamesamcl committed:
fix null pointer deref when no file hash is yet in the database

jamesamcl committed:
Also check status for null in case loading failed for some other reason

henrietteharmse committed:
Merge pull request #334 from EBISPOT/237-data-property-indexing

Index data properties in solr

jamesamcl committed:
Hopefully fix the cycles in jstree generation

jamesamcl committed:
Merge branch 'ebi-rebranded-merge2' into 3.x-dev

jamesamcl committed:
move custom web directory to fix static content 404s

jamesamcl committed:
fix typo in template (getConfig to getConfig())

jamesamcl committed:
Styling fixes, including prefixing of graph view HTML classes

jamesamcl committed:
New UI tweaks, including collapsible sections

jamesamcl committed:
Make tree views scrollable

jamesamcl committed:
Fix toggles appearing multiple times due to doubly included JS

jamesamcl committed:
Revert "Also check status for null in case loading failed for some other reason"

This reverts commit 175d6a5598c51ea44bb7b4801e161e4885b9ecaa.

jamesamcl committed:
Revert "fix null pointer deref when no file hash is yet in the database"

This reverts commit c2e855a1d06ab9de6d8937532afab652ea4499e9.

jamesamcl committed:
Revert "Hopefully fix the issue of ontologies not updating"

This reverts commit 93c53828455aa767f4a7cb8122d3c6ab713b7652.

jamesamcl committed:
Revert "Also check status for null in case loading failed for some other reason"

This reverts commit 175d6a5598c51ea44bb7b4801e161e4885b9ecaa.

jamesamcl committed:
Revert "fix null pointer deref when no file hash is yet in the database"

This reverts commit c2e855a1d06ab9de6d8937532afab652ea4499e9.

jamesamcl committed:
Revert "Hopefully fix the issue of ontologies not updating"

This reverts commit 93c53828455aa767f4a7cb8122d3c6ab713b7652.

henrietteharmse committed:
Upgrade of OWL API to 5.1.14

henrietteharmse committed:
Resolved HermiT and ELK reasoner related version conflicts

jamesamcl committed:
Fix for the issue of owl:Thing not being labelled as a Root in neo4j

An axiom subClassOf (someproperty some owl:Thing) could cause the owl:Thing
node to be created through the code path for "related nodes", which does
not set the Root label.

IMO the neo4j indexer is in need of refactoring to make it more explicit which
code paths are responsible for creating nodes, but meanwhile this minimal
fix simply ensures that all nodes are created before handling related nodes.

jamesamcl committed:
Merge remote-tracking branch 'origin/3.x-dev'

henrietteharmse committed:
Fix for Tomcat startup issue

henrietteharmse committed:
Merge 3.x-dev

jamesamcl committed:
Fix URIs in info box

jamesamcl committed:
Merge pull request #361 from EBISPOT/roots-fix

Fix for the issue of owl:Thing not being labelled as a Root in neo4j

jamesamcl committed:
Merge pull request #351 from EBISPOT/tree-cycle-fix

Hopefully fix the cycles in jstree generation

jamesamcl committed:
Fix disappearing annotations

jamesamcl committed:
Fix missing titles

jamesamcl committed:
Fix xrefs (#372)

jamesamcl committed:
Skip failing ontologies after multiple attempts

jamesamcl committed:
Reset load attempts to 0 after successful load

jamesamcl committed:
Merge pull request #366 from EBISPOT/new-ui-fixes

Fix URIs in info box

jamesamcl committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS

jamesamcl committed:
fix errors for terms with obo xrefs

jamesamcl committed:
Docker refactoring

henrietteharmse committed:
Added spreadsheet to request ontology to be added to OLS

jamesamcl committed:
fix tree issues (including #379)

jamesamcl committed:
Merge pull request #380 from EBISPOT/tree-fixes

fix tree issues (including #379)

jamesamcl committed:
add final docker compose instruction

henrietteharmse committed:
Merge pull request #375 from EBISPOT/skip-failing2

Skip failing ontologies after multiple attempts

jamesamcl committed:
fix invisible docs headers

jamesamcl committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS into 3.x-dev

jamesamcl committed:
Remove old docker files

jamesamcl committed:
Update docker instruction to include version :latest

jamesamcl committed:
Add .gitlab-ci.yml

jamesamcl committed:
Remove old gitlab config

jamesamcl committed:
update gitlab CI to build all images

jamesamcl committed:
limit ci to certain branches

jamesamcl committed:
Merge branch 'dev'

jamesamcl committed:
Further docker refactoring to fix issues with volumes

jamesamcl committed:
Add ols/obo configs back to ols-config-importer resources

jamesamcl committed:
Merge branch 'master' into 3.x-dev

jamesamcl committed:
fix only first annotation being shown

jamesamcl committed:
fix only first annotation being shown

henrietteharmse committed:
Added CPU and memory usages stats for indexing

henrietteharmse committed:
Minor updates

jamesamcl committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS into owlapi

henrietteharmse committed:
Added marker for resource usage logging

jamesamcl committed:
Merge pull request #392 from EBISPOT/owlapi

Owlapi

henrietteharmse committed:
Exclude owl api from hermit and elk

henrietteharmse committed:
Exclude owl api from hermit and elk

henrietteharmse committed:
Changed resource usage logging to info rather than debug.

henrietteharmse committed:
Merge 3.x-dev

henrietteharmse committed:
Fix for labels in imports issue

jamesamcl committed:
Merge pull request #393 from EBISPOT/3.x-dev

Fix for labels in imports issue

jamesamcl committed:
Use full IRI in getOWLAnnotationValueAsString if short form is empty (fixes #404)

johannes-darms committed:
Include SOLR config within docker image

jamesamcl committed:
Merge pull request #405 from EBISPOT/short-form-fix

Use full IRI in getOWLAnnotationValueAsString if short form is empty …

jamesamcl committed:
Merge pull request #408 from johannes-darms/patch-2

Include SOLR config within docker image

jamesamcl committed:
Remove solr build from docker compose file

henrietteharmse committed:
Workaround for owlapi EnititySearcher.getSuperProperties issue where super is inverse property.

jamesamcl committed:
Merge pull request #412 from EBISPOT/3.x-dev

3.x dev

jamesamcl committed:
Add powered by footer for debranded mode

jamesamcl committed:
Change contact us link to link to issue tracker

jamesamcl committed:
Test branch push restrictions

henrietteharmse committed:
Made ELK the default reasoner and removed use of Structural reasoner

jamesamcl committed:
Merge pull request #414 from EBISPOT/pseudo-reasoner

Made ELK the default reasoner and removed use of Structural reasoner

jamesamcl committed:
fix links/images not working in term property values

jamesamcl committed:
fix links/images not working in term property values

jamesamcl committed:
Merge branch 'master' into 3.x-dev

henrietteharmse committed:
Merge pull request #421 from EBISPOT/depictions-hotfix

Depictions hotfix

henrietteharmse committed:
Default reasoner set to ELK. Structural Reasoner used when ontology is inconsistent or has unsatisfiable classes. Ideally the pseudo reasoner should be used in these circumstances. For the test cases used efo, mondo - both the .obo and .owl and duo the Pseudo reasoner works well. However, for chebi it does not. The idea is to monitor use of the Structural Reasoner which will tell us which ontologies are incoherent. Then at a later stage we can test the Pseudo reasoner for these ontologies. Then there is also the possibility that the redesigned indexer may have less of a dependency on reasoners.

Commit:
Fix a typo in the .m2 path

henrietteharmse committed:
Defaults to structural reasoner when no reasoner is specified

henrietteharmse committed:
Commented out some logging

jamesamcl committed:
Merge pull request #432 from EBISPOT/pseudo-reasoner

Pseudo reasoner

henrietteharmse committed:
Merge pull request #433 from EBISPOT/pseudo-reasoner

Pseudo reasoner

jamesamcl committed:
Merge pull request #434 from EBISPOT/3.x-dev

3.x dev

henrietteharmse committed:
Merge pull request #429 from jiaola/master

Fix a typo in the .m2 path

jamesamcl committed:
fix ebi link in debranded footer

jamesamcl committed:
Fix formatting of ontology request form

henrietteharmse committed:
Merge pull request #436 from EBISPOT/3.x-dev

3.x dev

jamesamcl committed:
Reinstate solr config in compose file

jamesamcl committed:
Fix extracting short name from URNs

jamesamcl committed:
Merge branch '3.x-dev' of github.com:EBISPOT/OLS into 3.x-dev

henrietteharmse committed:
Merge pull request #441 from EBISPOT/3.x-dev

3.x dev

jamesamcl committed:
fix missing citations on term page

jamesamcl committed:
fix empty term annotations where the value is an IRI

jamesamcl committed:
Merge pull request #445 from EBISPOT/3.x-dev

3.x dev

henrietteharmse committed:
Defaulting reasoner to NONE rather than EL.

henrietteharmse committed:
Merge branch '3.x-dev' of https://github.com/EBISPOT/OLS into 3.x-dev

jamesamcl committed:
Merge pull request #447 from EBISPOT/3.x-dev

3.x dev

jamesamcl committed:
Add property information

jamesamcl committed:
Merge pull request #463 from EBISPOT/3.x-dev

Add property information
